--- a/chrome/browser/focus_flag_choices.h
+++ b/chrome/browser/focus_flag_choices.h
@@ -37,6 +37,8 @@ namespace focus {
   constexpr const char kFocusNoiseCpuCoresCommandLine[] =
       "focus-noise-hw-concurrency";
 
+  constexpr const char kMiddleClickAutoscrollCommandLine[] = "middle-click-autoscroll";
+
 }  // namespace focus
 
 #endif  /* CHROME_BROWSER_FOCUS_FLAG_CHOICES_H_ */
--- a/chrome/browser/focus_flag_entries.h
+++ b/chrome/browser/focus_flag_entries.h
@@ -36,4 +36,8 @@
      "Randomizes the number of cores returned by "
      "`navigator.hardwareConcurrency` in a reasonable range. Focus Browser flag.",
      kOsAll, FEATURE_VALUE_TYPE(blink::features::kFocusNoiseCpuCores)},
+    {focus::kMiddleClickAutoscrollCommandLine,
+     "Middle Click Autoscroll",
+     "Enables autoscroll on middle click. Focus Browser flag, Chromium feature.",
+     kOsDesktop, FEATURE_VALUE_TYPE(blink::features::kFocusMiddleClickAutoscroll)},
 #endif  /* CHROME_BROWSER_FOCUS_FLAG_ENTRIES_H_ */
--- a/third_party/blink/public/common/features.h
+++ b/third_party/blink/public/common/features.h
@@ -45,6 +45,8 @@ BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE
 BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kFocusNoiseAudio);
 BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kFocusNoiseCpuCores);
 
+BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kFocusMiddleClickAutoscroll);
+
 // Controls whether to include information about the page's open popup in
 // AIPageContent.
 BLINK_COMMON_EXPORT BASE_DECLARE_FEATURE(kAIPageContentIncludePopupWindows);
--- a/third_party/blink/common/features.cc
+++ b/third_party/blink/common/features.cc
@@ -19,6 +19,15 @@
 
 namespace blink::features {
 
+BASE_FEATURE(kFocusMiddleClickAutoscroll,
+             "FocusMiddleClickAutoscroll",
+#if BUILDFLAG(IS_WIN)
+             base::FEATURE_ENABLED_BY_DEFAULT
+#else
+             base::FEATURE_DISABLED_BY_DEFAULT
+#endif
+);
+
 BASE_FEATURE(kDisableLinkDrag, "DisableLinkDrag", base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kReducedSystemInfo, "ReducedSystemInfo", base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kRemoveClientHints, "RemoveClientHints", base::FEATURE_DISABLED_BY_DEFAULT);
--- a/third_party/blink/renderer/core/exported/web_view_impl.cc
+++ b/third_party/blink/renderer/core/exported/web_view_impl.cc
@@ -1874,9 +1874,9 @@ void WebView::ApplyWebPreferences(const
       prefs.default_maximum_page_scale_factor);
 #endif
 
-#if BUILDFLAG(IS_WIN)
-  RuntimeEnabledFeatures::SetMiddleClickAutoscrollEnabled(true);
-#endif
+  RuntimeEnabledFeatures::SetMiddleClickAutoscrollEnabled(
+    base::FeatureList::IsEnabled(features::kFocusMiddleClickAutoscroll)
+  );
 
   RuntimeEnabledFeatures::SetTranslateServiceEnabled(
       prefs.translate_service_available);
