diff --git a/chrome/browser/ui/webui/settings/brave_import_data_handler.cc b/chrome/browser/ui/webui/settings/focus_import_data_handler.cc
similarity index 90%
rename from chrome/browser/ui/webui/settings/brave_import_data_handler.cc
rename to chrome/browser/ui/webui/settings/focus_import_data_handler.cc
--- a/chrome/browser/ui/webui/settings/brave_import_data_handler.cc
+++ b/chrome/browser/ui/webui/settings/focus_import_data_handler.cc
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "chrome/browser/ui/webui/settings/brave_import_data_handler.h"
+#include "chrome/browser/ui/webui/settings/focus_import_data_handler.h"
 
 #include <memory>
 #include <string>
@@ -64,10 +64,10 @@
 
 namespace settings {
 
-BraveImportDataHandler::BraveImportDataHandler() = default;
-BraveImportDataHandler::~BraveImportDataHandler() = default;
+FocusImportDataHandler::FocusImportDataHandler() = default;
+FocusImportDataHandler::~FocusImportDataHandler() = default;
 
-void BraveImportDataHandler::StartImport(
+void FocusImportDataHandler::StartImport(
     const user_data_importer::SourceProfile& source_profile,
     uint16_t imported_items) {
   if (!imported_items) {
@@ -99,7 +99,7 @@
 #if BUILDFLAG(IS_MAC)
   CheckDiskAccess(imported_items, source_profile.source_path,
                   source_profile.importer_type,
-                  base::BindOnce(&BraveImportDataHandler::StartImportImpl,
+                  base::BindOnce(&FocusImportDataHandler::StartImportImpl,
                                  weak_factory_.GetWeakPtr(), source_profile,
                                  imported_items, profile));
 #else
@@ -107,7 +107,7 @@
 #endif
 }
 
-void BraveImportDataHandler::StartImportImpl(
+void FocusImportDataHandler::StartImportImpl(
     const user_data_importer::SourceProfile& source_profile,
     uint16_t imported_items,
     Profile* profile) {
@@ -145,9 +145,9 @@
   // Using weak pointers because it destroys itself when finshed.
   auto* importer_host = new BraveExternalProcessImporterHost();
   import_observers_[source_profile.source_path] =
-      std::make_unique<BraveImporterObserver>(
+      std::make_unique<FocusImporterObserver>(
           importer_host, source_profile, imported_items,
-          base::BindRepeating(&BraveImportDataHandler::NotifyImportProgress,
+          base::BindRepeating(&FocusImportDataHandler::NotifyImportProgress,
                               weak_factory_.GetWeakPtr()));
 
   importer_host->set_parent_window(web_contents->GetTopLevelNativeWindow());
@@ -157,7 +157,7 @@
                                      new ProfileWriter(profile));
 }
 
-void BraveImportDataHandler::NotifyImportProgress(
+void FocusImportDataHandler::NotifyImportProgress(
     const user_data_importer::SourceProfile& source_profile,
     const base::DictValue& info) {
   const std::string* event = info.FindString("event");
@@ -168,16 +168,16 @@
     import_did_succeed_ = true;
   } else if (*event == "ImportEnded") {
     content::GetUIThreadTaskRunner({})->PostTask(
-        FROM_HERE, base::BindOnce(&BraveImportDataHandler::OnImportEnded,
+        FROM_HERE, base::BindOnce(&FocusImportDataHandler::OnImportEnded,
                                   weak_factory_.GetWeakPtr(), source_profile));
   }
 }
 
-void BraveImportDataHandler::HandleImportData(const base::ListValue& args) {
+void FocusImportDataHandler::HandleImportData(const base::ListValue& args) {
   ImportDataHandler::HandleImportData(args);
 }
 
-void BraveImportDataHandler::OnImportEnded(
+void FocusImportDataHandler::OnImportEnded(
     const user_data_importer::SourceProfile& source_profile) {
   import_observers_.erase(source_profile.source_path);
   FireWebUIListener("import-data-status-changed",
@@ -185,7 +185,7 @@
                                                     : kImportStatusFailed));
 }
 
-void BraveImportDataHandler::OnJavascriptDisallowed() {
+void FocusImportDataHandler::OnJavascriptDisallowed() {
   weak_factory_.InvalidateWeakPtrs();
   ImportDataHandler::OnJavascriptDisallowed();
 
@@ -195,12 +195,12 @@
 }
 
 const user_data_importer::SourceProfile&
-BraveImportDataHandler::GetSourceProfileAt(int browser_index) {
+FocusImportDataHandler::GetSourceProfileAt(int browser_index) {
   return importer_list_->GetSourceProfileAt(browser_index);
 }
 
 #if BUILDFLAG(IS_MAC)
-void BraveImportDataHandler::CheckDiskAccess(
+void FocusImportDataHandler::CheckDiskAccess(
     uint16_t imported_items,
     base::FilePath source_path,
     user_data_importer::ImporterType importer_type,
@@ -215,7 +215,7 @@
     base::ThreadPool::PostTaskAndReplyWithResult(
         FROM_HERE, {base::MayBlock()},
         base::BindOnce(&HasProperDiskAccessPermission, imported_items),
-        base::BindOnce(&BraveImportDataHandler::OnGetDiskAccessPermission,
+        base::BindOnce(&FocusImportDataHandler::OnGetDiskAccessPermission,
                        weak_factory_.GetWeakPtr(), std::move(callback),
                        source_path));
     return;
@@ -223,7 +223,7 @@
   std::move(callback).Run();
 }
 
-void BraveImportDataHandler::OnGetDiskAccessPermission(
+void FocusImportDataHandler::OnGetDiskAccessPermission(
     ContinueImportCallback callback,
     base::FilePath source_path,
     bool allowed) {
@@ -246,7 +246,7 @@
   std::move(callback).Run();
 }
 
-void BraveImportDataHandler::DidStopLoading() {
+void FocusImportDataHandler::DidStopLoading() {
   Observe(nullptr);
 
   if (!guide_dialog_is_requested_) {
diff --git a/chrome/browser/ui/webui/settings/brave_import_data_handler.h b/chrome/browser/ui/webui/settings/focus_import_data_handler.h
similarity index 90%
rename from chrome/browser/ui/webui/settings/brave_import_data_handler.h
rename to chrome/browser/ui/webui/settings/focus_import_data_handler.h
--- a/chrome/browser/ui/webui/settings/brave_import_data_handler.h
+++ b/chrome/browser/ui/webui/settings/focus_import_data_handler.h
@@ -3,24 +3,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORT_DATA_HANDLER_H_
-#define BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORT_DATA_HANDLER_H_
+#ifndef CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORT_DATA_HANDLER_H_
+#define CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORT_DATA_HANDLER_H_
 
 #include <memory>
 #include <unordered_map>
 
 #include "base/memory/weak_ptr.h"
-#include "chrome/browser/ui/webui/settings/brave_importer_observer.h"
+#include "chrome/browser/ui/webui/settings/focus_importer_observer.h"
 #include "build/build_config.h"
 #include "chrome/browser/ui/webui/settings/import_data_handler.h"
 #include "content/public/browser/web_contents_observer.h"
 
 namespace settings {
 
-// This class checks whether Brave has full disk access permission to import
-// safari data on macOS. ImportDataHandler::StartImport() will be run after
-// checking disk access permission. If Brave doesn't have that permission, this
-// will launch tab modal dialog to notify users about this lack of permission.
+// This class checks whether Focus Browser has full disk access permission to
+// import Safari data on macOS. ImportDataHandler::StartImport() will be run
+// after checking disk access permission. If Focus Browser doesn't have that
+// permission, this will launch a tab modal dialog to notify the user.
 
 // We should display tab modal dialog after import dialog is closed from webui.
 // To do that, this observes web contents to launch dialog after import dialog
@@ -29,14 +29,14 @@
 // new navigation start and tab is newly loaded for closing webui import dialog.
 // The reason why native tab modal dialog is used here is to avoid modifying
 // upstream import html/js source code.
-class BraveImportDataHandler : public ImportDataHandler,
-                                      content::WebContentsObserver {
+class FocusImportDataHandler : public ImportDataHandler,
+                               content::WebContentsObserver {
  public:
-  BraveImportDataHandler();
-  ~BraveImportDataHandler() override;
+  FocusImportDataHandler();
+  ~FocusImportDataHandler() override;
 
-  BraveImportDataHandler(const BraveImportDataHandler&) = delete;
-  BraveImportDataHandler& operator=(const BraveImportDataHandler&) = delete;
+  FocusImportDataHandler(const FocusImportDataHandler&) = delete;
+  FocusImportDataHandler& operator=(const FocusImportDataHandler&) = delete;
 
  protected:
   using ContinueImportCallback = base::OnceCallback<void()>;
@@ -76,11 +77,11 @@
   bool guide_dialog_is_requested_ = false;
 #endif
  private:
-  std::unordered_map<base::FilePath, std::unique_ptr<BraveImporterObserver>>
+  std::unordered_map<base::FilePath, std::unique_ptr<FocusImporterObserver>>
       import_observers_;
-  base::WeakPtrFactory<BraveImportDataHandler> weak_factory_{this};
+  base::WeakPtrFactory<FocusImportDataHandler> weak_factory_{this};
 };
 
 }  // namespace settings
 
-#endif  // BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORT_DATA_HANDLER_H_
+#endif  // CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORT_DATA_HANDLER_H_
diff --git a/chrome/browser/ui/webui/settings/brave_importer_observer.cc b/chrome/browser/ui/webui/settings/focus_importer_observer.cc
similarity index 90%
rename from chrome/browser/ui/webui/settings/brave_importer_observer.cc
rename to chrome/browser/ui/webui/settings/focus_importer_observer.cc
--- a/chrome/browser/ui/webui/settings/brave_importer_observer.cc
+++ b/chrome/browser/ui/webui/settings/focus_importer_observer.cc
@@ -3,14 +3,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "chrome/browser/ui/webui/settings/brave_importer_observer.h"
+#include "chrome/browser/ui/webui/settings/focus_importer_observer.h"
 
 #include <utility>
 
 #include "base/check.h"
 #include "chrome/browser/importer/external_process_importer_host.h"
 
-BraveImporterObserver::BraveImporterObserver(
+FocusImporterObserver::FocusImporterObserver(
     ExternalProcessImporterHost* importer_host,
     const user_data_importer::SourceProfile& source_profile,
     uint16_t imported_items,
@@ -23,13 +23,13 @@
   importer_host->set_observer(this);
 }
 
-BraveImporterObserver::~BraveImporterObserver() {
+FocusImporterObserver::~FocusImporterObserver() {
   if (importer_host_)
     importer_host_->set_observer(nullptr);
 }
 
 // user_data_importer::ImporterProgressObserver:
-void BraveImporterObserver::ImportStarted() {
+void FocusImporterObserver::ImportStarted() {
   if (import_started_called_)
     return;
   import_started_called_ = true;
@@ -41,7 +41,7 @@
   callback_.Run(source_profile_, data);
 }
 
-void BraveImporterObserver::ImportItemStarted(
+void FocusImporterObserver::ImportItemStarted(
     user_data_importer::ImportItem item) {
   base::DictValue data;
   data.Set("importer_name", source_profile_.importer_name);
@@ -52,7 +52,7 @@
   callback_.Run(source_profile_, data);
 }
 
-void BraveImporterObserver::ImportItemEnded(
+void FocusImporterObserver::ImportItemEnded(
     user_data_importer::ImportItem item) {
   base::DictValue data;
   data.Set("importer_name", source_profile_.importer_name);
@@ -63,7 +63,7 @@
   callback_.Run(source_profile_, data);
 }
 
-void BraveImporterObserver::ImportEnded() {
+void FocusImporterObserver::ImportEnded() {
   base::DictValue data;
   data.Set("importer_name", source_profile_.importer_name);
   data.Set("importer_type", source_profile_.importer_type);
@@ -80,6 +80,6 @@
 }
 
 ExternalProcessImporterHost*
-BraveImporterObserver::GetImporterHostForTesting() {
+FocusImporterObserver::GetImporterHostForTesting() {
   return importer_host_.get();
 }
diff --git a/chrome/browser/ui/webui/settings/brave_importer_observer.h b/chrome/browser/ui/webui/settings/focus_importer_observer.h
similarity index 90%
rename from chrome/browser/ui/webui/settings/brave_importer_observer.h
rename to chrome/browser/ui/webui/settings/focus_importer_observer.h
--- a/chrome/browser/ui/webui/settings/brave_importer_observer.h
+++ b/chrome/browser/ui/webui/settings/focus_importer_observer.h
@@ -3,8 +3,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORTER_OBSERVER_H_
-#define BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORTER_OBSERVER_H_
+#ifndef CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORTER_OBSERVER_H_
+#define CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORTER_OBSERVER_H_
 
 #include "base/functional/callback.h"
 #include "base/gtest_prod_util.h"
@@ -14,17 +14,17 @@
 
 class ExternalProcessImporterHost;
 
-class BraveImporterObserver : public importer::ImporterProgressObserver {
+class FocusImporterObserver : public importer::ImporterProgressObserver {
  public:
   using ReportProgressCallback = base::RepeatingCallback<void(
       const user_data_importer::SourceProfile& source_profile,
       const base::DictValue&)>;
 
-  BraveImporterObserver(ExternalProcessImporterHost* host,
+  FocusImporterObserver(ExternalProcessImporterHost* host,
                         const user_data_importer::SourceProfile& source_profile,
                         uint16_t imported_items,
                         ReportProgressCallback callback);
-  ~BraveImporterObserver() override;
+  ~FocusImporterObserver() override;
 
   void ImportStarted() override;
   void ImportItemStarted(user_data_importer::ImportItem item) override;
@@ -32,7 +32,7 @@
   void ImportEnded() override;
 
  private:
-  FRIEND_TEST_ALL_PREFIXES(BraveImporterObserverUnitTest, ImportEvents);
+  FRIEND_TEST_ALL_PREFIXES(FocusImporterObserverUnitTest, ImportEvents);
 
   ExternalProcessImporterHost* GetImporterHostForTesting();
 
@@ -47,4 +47,4 @@
   raw_ptr<ExternalProcessImporterHost> importer_host_;  // weak
 };
 
-#endif  // BRAVE_BROWSER_UI_WEBUI_SETTINGS_BRAVE_IMPORTER_OBSERVER_H_
+#endif  // CHROME_BROWSER_UI_WEBUI_SETTINGS_FOCUS_IMPORTER_OBSERVER_H_
diff --git a/chrome/browser/ui/webui/settings/import_data_handler.h b/chrome/browser/ui/webui/settings/import_data_handler.h
--- a/chrome/browser/ui/webui/settings/import_data_handler.h
+++ b/chrome/browser/ui/webui/settings/import_data_handler.h
@@ -36,7 +36,7 @@
   void RegisterMessages() override;
   void OnJavascriptAllowed() override {}
   void OnJavascriptDisallowed() override;
-  friend class BraveImportDataHandler;
+  friend class FocusImportDataHandler;
  private:
   virtual void StartImport(const user_data_importer::SourceProfile& source_profile,
                    uint16_t imported_items);
diff --git a/chrome/browser/ui/webui/settings/settings_ui.cc b/chrome/browser/ui/webui/settings/settings_ui.cc
--- a/chrome/browser/ui/webui/settings/settings_ui.cc
+++ b/chrome/browser/ui/webui/settings/settings_ui.cc
@@ -64,7 +64,7 @@
 #include "chrome/browser/ui/webui/settings/about_handler.h"
 #include "chrome/browser/ui/webui/settings/accessibility_main_handler.h"
 #include "chrome/browser/ui/webui/settings/appearance_handler.h"
-#include "chrome/browser/ui/webui/settings/brave_import_data_handler.h"
+#include "chrome/browser/ui/webui/settings/focus_import_data_handler.h"
 #include "chrome/browser/ui/webui/settings/browser_lifetime_handler.h"
 #include "chrome/browser/ui/webui/settings/browser_shortcuts_handler.h"
 #include "chrome/browser/ui/webui/settings/downloads_handler.h"
@@ -255,7 +255,7 @@
   AddSettingsPageUIHandler(std::make_unique<DownloadsHandler>(profile));
   AddSettingsPageUIHandler(std::make_unique<ExtensionControlHandler>());
   AddSettingsPageUIHandler(std::make_unique<FontHandler>(profile));
-  AddSettingsPageUIHandler(std::make_unique<BraveImportDataHandler>());
+  AddSettingsPageUIHandler(std::make_unique<FocusImportDataHandler>());
   AddSettingsPageUIHandler(std::make_unique<HatsHandler>());
 
 #if BUILDFLAG(IS_WIN)
diff --git a/chrome/browser/ui/webui/onboarding/onboarding_page_ui.cc b/chrome/browser/ui/webui/onboarding/onboarding_page_ui.cc
--- a/chrome/browser/ui/webui/onboarding/onboarding_page_ui.cc
+++ b/chrome/browser/ui/webui/onboarding/onboarding_page_ui.cc
@@ -8,7 +8,7 @@
 
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/webui/onboarding/onboarding_handler.h"
-#include "chrome/browser/ui/webui/settings/brave_import_data_handler.h"
+#include "chrome/browser/ui/webui/settings/focus_import_data_handler.h"
 #include "chrome/browser/ui/webui/settings/search_engines_handler.h"
 #include "chrome/browser/ui/webui/settings/settings_default_browser_handler.h"
 #include "chrome/common/webui_url_constants.h"
@@ -52,7 +52,7 @@
   web_ui->AddMessageHandler(
       std::make_unique<settings::DefaultBrowserHandler>());
   web_ui->AddMessageHandler(
-      std::make_unique<settings::BraveImportDataHandler>());
+      std::make_unique<settings::FocusImportDataHandler>());
 }
 
 FocusOnboardingPage::~FocusOnboardingPage() = default;
diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1284,10 +1284,10 @@
       "webui/settings/accessibility_main_handler.h",
       "webui/settings/appearance_handler.cc",
       "webui/settings/appearance_handler.h",
-      "webui/settings/brave_import_data_handler.cc",
-      "webui/settings/brave_import_data_handler.h",
-      "webui/settings/brave_importer_observer.cc",
-      "webui/settings/brave_importer_observer.h",
+      "webui/settings/focus_import_data_handler.cc",
+      "webui/settings/focus_import_data_handler.h",
+      "webui/settings/focus_importer_observer.cc",
+      "webui/settings/focus_importer_observer.h",
       "webui/settings/browser_lifetime_handler.cc",
       "webui/settings/browser_lifetime_handler.h",
       "webui/settings/captions_handler.cc",
diff --git a/chrome/browser/importer/brave_external_process_importer_host.h b/chrome/browser/importer/brave_external_process_importer_host.h
--- a/chrome/browser/importer/brave_external_process_importer_host.h
+++ b/chrome/browser/importer/brave_external_process_importer_host.h
@@ -33,8 +33,8 @@
   friend class ExternalProcessImporterHost;
   friend class BraveExternalProcessImporterHostUnitTest;
 
-  FRIEND_TEST_ALL_PREFIXES(BraveImporterObserverUnitTest, ImportEvents);
-  FRIEND_TEST_ALL_PREFIXES(BraveImporterObserverUnitTest, DestroyObserverEarly);
+  FRIEND_TEST_ALL_PREFIXES(FocusImporterObserverUnitTest, ImportEvents);
+  FRIEND_TEST_ALL_PREFIXES(FocusImporterObserverUnitTest, DestroyObserverEarly);
 
   ~BraveExternalProcessImporterHost() override;
 
