--- a/chrome/browser/ui/views/toolbar/BUILD.gn
+++ b/chrome/browser/ui/views/toolbar/BUILD.gn
@@ -17,6 +17,7 @@ source_set("toolbar") {
     "chrome_labs/chrome_labs_coordinator.h",
     "chrome_labs/chrome_labs_item_view.h",
     "chrome_labs/chrome_labs_view_controller.h",
+    "focus_youtube_bubble_view.h",
     "home_button.h",
     "overflow_button.h",
     "pinned_action_toolbar_button.h",
@@ -113,6 +114,7 @@ source_set("impl") {
     "chrome_labs/chrome_labs_coordinator.cc",
     "chrome_labs/chrome_labs_item_view.cc",
     "chrome_labs/chrome_labs_view_controller.cc",
+    "focus_youtube_bubble_view.cc",
     "home_button.cc",
     "overflow_button.cc",
     "pinned_action_toolbar_button.cc",
@@ -222,5 +224,6 @@ source_set("impl") {
     "//components/autofill/core/common:features",
     "//components/feature_engagement/public",
+    "//components/focus_services",
     "//components/omnibox/browser",
     "//components/password_manager/content/common",
     "//components/performance_manager",
@@ -246,4 +249,5 @@ source_set("impl") {
     "//content/public/browser",
     "//extensions/browser/api/feedback_private",
+    "//extensions/browser/api/storage",
     "//extensions/common",
     "//media",
