diff --git a/chrome/app/resources/chromium_strings_ru.xtb b/chrome/app/resources/chromium_strings_ru.xtb
--- a/chrome/app/resources/chromium_strings_ru.xtb
+++ b/chrome/app/resources/chromium_strings_ru.xtb
@@ -487,6 +487,7 @@
 <translation id="5608522662649299320">Чтобы продолжить, войдите в Focus Browser с аккаунтом <ph name="EMAIL" />. Это необходимо, чтобы применить правила организации.</translation>
 <translation id="3609072104461976221">Логотип Focus Browser на экране компьютера.</translation>
 <translation id="4408237449933814174">Focus Browser – браузер, который открывает веб-страницы и приложения за секунды. Он быстр, стабилен в работе и прост в использовании. Встроенная в Focus Browser защита от мошенничества и вредоносного ПО обеспечивает максимально безопасную работу в Интернете.</translation>
+<translation id="7309892594118311538">Focus Browser — быстрый и стабильный браузер для сосредоточенной работы. В него встроены FocusBlock, FocusYoutube и инструменты управления отвлекающими элементами.</translation>
 <translation id="7436012071236786215">Чтобы всегда иметь доступ к данным Focus Browser и возможность сохранять их в аккаунте Google, подтвердите свою личность.</translation>
 <translation id="7028768937485579049">Повысится скорость Focus Browser и вероятность корректной работы функций, использующих JavaScript</translation>
 <translation id="6436260184216827876">Поиск по изображениям недоступен. Повторите попытку позже.</translation>
