--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -654,11 +654,10 @@ if (is_win) {
     ]
     outputs = [ "{{bundle_resources_dir}}/{{source_file_part}}" ]
     public_deps = [
       ":chrome_app_icon",
       ":chrome_app_strings",
-      ":chrome_asset_catalog",
       "//components/policy:chrome_manifest_bundle",
     ]
   }
 
   bundle_data("chrome_versioned_bundle_data") {
@@ -871,11 +870,10 @@ if (is_win) {
     helper_name_suffix = alert_helper_params[2]
     helper_bundle_id_suffix = alert_helper_params[1]
     info_plist_target = ":chrome_helper_app_alerts_plist"
     deps = [
       ":chrome_app_icon",
-      ":chrome_asset_catalog",
       ":chrome_helper_app_alerts_resources",
     ]
   }
 
   if (verify_dynamic_libraries) {
--- a/chrome/app/app-Info.plist
+++ b/chrome/app/app-Info.plist
@@ -149,8 +149,6 @@
 	<string>${EXECUTABLE_NAME}</string>
 	<key>CFBundleIconFile</key>
 	<string>app.icns</string>
-	<key>CFBundleIconName</key>
-	<string>AppIcon</string>
 	<key>CFBundleIdentifier</key>
 	<string>${CHROMIUM_BUNDLE_ID}</string>
 	<key>CFBundleInfoDictionaryVersion</key>
--- a/chrome/app/helper-alerts-Info.plist
+++ b/chrome/app/helper-alerts-Info.plist
@@ -6,8 +6,6 @@
 	<string>${CHROMIUM_BUNDLE_ID}.framework.AlertNotificationService</string>
 	<key>CFBundleIconFile</key>
 	<string>app.icns</string>
-	<key>CFBundleIconName</key>
-	<string>AppIcon</string>
 	<key>NSUserNotificationAlertStyle</key>
 	<string>alert</string>
 </dict>
