diff --git a/chrome/installer/mac/signing/parts.py b/chrome/installer/mac/signing/parts.py
--- a/chrome/installer/mac/signing/parts.py
+++ b/chrome/installer/mac/signing/parts.py
@@ -123,7 +123,6 @@ def get_parts(config):
     dylibs = [
         'libEGL.dylib',
         'libGLESv2.dylib',
-        'libvk_swiftshader.dylib',
     ]
     if config.is_chrome_branded():
         dylibs.append('liboptimization_guide_internal.dylib')
